test/common.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const fs = require('fs');
+
+const REPORT_SECTIONS = [
+  'NodeReport',
+  'JavaScript Stack Trace',
+  'JavaScript Heap',
+  'System Information'
+];
+
+exports.locate = (pid) => {
+  var files = fs.readdirSync(".");
+  for (var i = 0; i < files.length; i++) {
+    if (files[i].substring(0, 10) == 'NodeReport' && files[i].indexOf(pid) != -1) {
+      return files[i];
+    }
+  }
+  return;
```

Comment:
This was how it was in the original [autorun.js](https://github.com/nodejs/nodereport/blob/v1.0.4/test/autorun.js#L106). I'll get rid of it.